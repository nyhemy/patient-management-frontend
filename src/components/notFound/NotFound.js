import React from 'react';
import styles from './NotFound.module.css';

/**
 * Component that handles 404s
 */
const NotFound = () => (
  <h3 className={styles.center}>404 Not Found</h3>
);

export default NotFound;
