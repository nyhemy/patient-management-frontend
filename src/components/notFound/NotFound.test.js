import React from 'react';
import { render } from '@testing-library/react';
import { unmountComponentAtNode } from 'react-dom';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';
import '@testing-library/jest-dom/extend-expect';

test('renders w/o crashing', () => {
  const div = document.createElement('div');
  render(<NotFound />, div);
  unmountComponentAtNode(div);
});

test('component matches snapshot', () => {
  const component = renderer.create(<NotFound />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
