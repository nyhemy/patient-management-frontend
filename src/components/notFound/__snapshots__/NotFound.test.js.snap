// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component matches snapshot 1`] = `
<h3
  className="center"
>
  404 Not Found
</h3>
`;
